BlueprintUICommonControls/Sources/Image.swift

```diff
@@ -15,6 +15,9 @@ public struct Image: Element {
     /// not precisely match the size that the element is assigned.
     public var contentMode: ContentMode
 
+    /// Whether to override the user interface style, `nil` is the system style.
+    public var overrideUserInterfaceStyle: UIUserInterfaceStyle?
```

Comment:
Interesting! I wonder if exposing this as a view-backed element that sets `overrideUserInterfaceStyle` on itself makes more sense so any views downstream get this? Eg:

```swift
myElement
 .overrideUserInterfaceStyle(.light)
```

Reply:
@kyleve you mean like this? (check the latest changes)